ocfweb/settings.py

```diff
@@ -16,6 +16,7 @@
     'dev.ocf.berkeley.edu',
     'dev-www.ocf.berkeley.edu',
     'ocfweb.ocf.berkeley.edu',
+    'supernova.ocf.berkeley.edu'
```

Comment:
this shouldn't be necessary since it's not enforced in debug mode, can you talk more about how you're hitting this?

Reply:
Agreed -- fixed this!